CompositionRoot/A.swift

```diff
@@ -4,15 +4,20 @@ protocol AProtocol {
 
 class A: AProtocol {
     
-    private let applicationContext: ApplicationContext
-    init(applicationContext: ApplicationContext) {
```

Comment:
It's not necessary to inject the whole `ApplicationContext` object. We just use two of them.